generateTestAddresses.js

```diff
@@ -0,0 +1,41 @@
+const { Wallet } = require("ethers");
+const fs = require('fs')
+const outputfile = 'genesisAddresses.json'
+const outputfileKeys = 'genesisKeys.txt'
+
+
+for (i = 1; i <= 1000; i++) {
```

Comment:
Please add these to README.md, at least some kind of reference how to take them into use (I see they're the keys 0x0000...00nnn)

Reply:
this is not really needed here, I used them once to generate the addresses to put into the genesisblocks, and the keys I would add to the network monorepo to use for the tests, but there is actually no real need to run this again. I just didn't know where to put this, maybe it would be better in the open-ethereum-poa? what do you think?

Reply:
aha, ok... hmm, I think it should be part of the config package, once we have such thing

Reply:
README should still be updated I think, but I can do that